ganga/GangaCore/__init__.py

```diff
@@ -29,7 +29,7 @@ def getLCGRootPath():
 # ------------------------------------------------
 # store Ganga version based on new git tag for this file
 _gangaVersion = '7.1.9'
-_development = True
+_development = False
```

Comment:
Please don't change this.

Reply:
Sorry, I did not change it myself. I was modifying original files from the ganga-7.1.9 release and that's how they came.
Can you fix it or do you want me to do something about it?